save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -81,16 +95,32 @@ class WarnPlugin(testConfig: TestConfig) : Plugin(testConfig) {
         )
     }
 
+    /**
+     * @param path
+     */
+    internal fun createTestFile(path: Path) {
+        val tmpDir = (FileSystem.SYSTEM_TEMPORARY_DIRECTORY / WarnPlugin::class.simpleName!!)
+
+        if (fs.exists(tmpDir)) {
+            fs.deleteRecursively(tmpDir)
+        }
+        fs.createDirectory(tmpDir)
+
+        fs.write(fs.createFile(tmpDir / "test_file")) {
+            fs.readLines(path).forEach {
+                if (!it.contains("// ;warn:")) {
```

Comment:
Do we have a pattern for this somewhere? There could be languages (like python) where `//` isn't a comment sign